code/modules/surgery/organs/organ_internal.dm

```diff
@@ -126,6 +126,51 @@
 /obj/item/organ/proc/OnEatFrom(eater, feeder)
 	useable = FALSE //You can't use it anymore after eating it you spaztic
 
+/* 
+ * On accidental consumption, cause organ damage and check if they like eating organs
+ */
+/obj/item/organ/on_accidental_consumption(mob/living/carbon/M, mob/living/carbon/user, obj/item/source_item, discover_after = TRUE)
```

Comment:
does this run when you try to eat an organ normally/outside of another food item?

Reply:
organ eating is handled by the edible component, so there's no overlap between this proc and that